### PR TITLE
License fix

### PR DESCRIPTION
**Pull request by PaulKMueller**
Checklist
* [x] Used a fork of the feedstock to propose changes
* [x] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/conda_smithy.html#how-to-re-render ) with the latest `conda-smithy`
* [ ] Ensured the license file is being packaged.

This PR changes the package's license format to comply with the `SPDX` specification (see [here](https://spdx.org/licenses/BSD-3-Clause.html)).

### CODE REVIEW

**Comment by Carreau** [comment]
Is that something that should be fixed upstream in https://github.com/ipython/ipython_genutils ?

**Comment by Carreau** [comment]
I've tried to re-add pins, but >=2.7

File: recipe/meta.yaml [hunk 1]
```diff
@@ -29,7 +29,9 @@ test:
 
 about:
   home: http://ipython.org
-  license: BSD 3-Clause
+  license: BSD-3-Clause
+  license_family: BSD
+  license_file: LICENSE
```
Comment (Carreau) [inline recipe/meta.yaml]:
```suggestion
  license_file: LICENSE.txt
```

File: recipe/meta.yaml [hunk 2]
```diff
@@ -29,7 +29,9 @@ test:
 
 about:
   home: http://ipython.org
-  license: BSD 3-Clause
+  license: BSD-3-Clause
+  license_family: BSD
+  license_file: LICENSE.txt
```
Comment (jakirkham) [inline recipe/meta.yaml]: Looking [at upstream]( https://github.com/ipython/ipython_genutils/blob/0.2.0/COPYING.md ), this appears to be the license filename

```suggestion
  license_file: COPYING.md
```

Comment (Carreau) [reply recipe/meta.yaml]: Ah, I assumed it referred to the file in the feedstock. Thanks.

**Pull request merged**